stetho/src/main/java/com/facebook/stetho/inspector/elements/AbstractChainedDescriptor.java

```diff
@@ -140,12 +140,30 @@ protected void onSetAttributesAsText(E element, String text) {
   }
 
   @Override
-  public final void getStyles(E element, StyleAccumulator accumulator) {
-    mSuper.getStyles(element, accumulator);
-    onGetStyles(element, accumulator);
+  public final void getStyleRules(E element, StyleRuleAccumulator accumulator) {
+    mSuper.getStyleRules(element, accumulator);
+    onGetStyleRules(element, accumulator);
   }
 
-  protected void onGetStyles(E element, StyleAccumulator accumulator) {
+  protected void onGetStyleRules(E element, StyleRuleAccumulator accumulator) {
+  }
+
+  @Override
+  public final void getStyles(E element, String ruleName, StyleAccumulator accumulator) {
+    mSuper.getStyles(element, ruleName, accumulator);
+    onGetStyles(element, ruleName, accumulator);
+  }
+
+  protected void onGetStyles(E element, String ruleName, StyleAccumulator accumulator) {
+  }
+
+  @Override
+  public final void setStyle(E element, String ruleName, String name, String value) {
+    mSuper.setStyle(element, ruleName, name, value);
```

Comment:
I can definetly see `View` handling some style such as `width` and `height` and `View` subclasses such as `LinearLayout` handling other styles such as `orientation`.